javascript/javascript1/week2/script.js

```diff
@@ -61,4 +61,25 @@ function getEventWeekday(days) {
 
 const result = getEventWeekday(2);
 
-console.log(result);
\ No newline at end of file
+console.log(result);
+
+
+// Weather wear
```

Comment:
This very nice and clean. I want to encourage you to use switch case in here , if you want to practice. otherwise totally fine.